cl_dll/hud_spectator.cpp

```diff
@@ -1255,8 +1273,8 @@ bool CHudSpectator::IsActivePlayer(cl_entity_t * ent)
 
 bool CHudSpectator::ParseOverviewFile( )
 {
-	char filename[255];
-	char levelname[255];
+	std::string filename;
+	std::string levelname;
 	char token[1024];
 	float height;
 	
```

Comment:
`memset( &m_OverviewData, 0, sizeof(m_OverviewData));` just below this line is UB, especially so since `Overview` is non-POD. Remove it and move the default value setting from here to the default constructor, and just call the default constructor (`m_OverviewData = Overview();`).